torch/utils/tensorboard/_embedding.py

```diff
@@ -42,31 +44,29 @@ def make_sprite(label_img, save_path):
         im.save(buf, format="PNG")
         im_bytes = buf.getvalue()
 
-    fs = tf.io.gfile.get_filesystem(save_path)
-    fs.write(fs.join(save_path, "sprite.png"), im_bytes, binary_mode=True)
+    with open(os.path.join(save_path, "sprite.png"), "wb") as f:
+        f.write(tf.compat.as_bytes(im_bytes))
 
 
 def get_embedding_info(metadata, label_img, filesys, subdir, global_step, tag):
```

Comment:
Maybe remove now unused `filesys` from here and any callers?

Reply:
No tensorboard is being used, only the fpython file system is being used to create the necessary files, os and tensorflow are being used.